test/unit/cli/webroot-locator-test.js

```diff
@@ -0,0 +1,73 @@
+var proxyquire = require('proxyquire').noCallThru()
+var simple = require('simple-mock')
+var test = require('tap').test
+var path = require('path')
+
+var fsMock = {
+  'existsSync': simple.spy(),
+  'statSync': simple.spy()
+}
+
+var statsMock = {
+  'isDirectory': simple.spy()
+}
+
+test('webroot locator', function (group) {
+  var webrootLocator = proxyquire('../../../cli/webroot-locator',
+    {fs: fsMock})
+
+  group.afterEach(function (done) {
+    fsMock.existsSync.actions.length = 0
```

Comment:
Comment moved from below: It's shorthand for emptying an array. I can change this to be a bit more explicit if required?